spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/Trade.java

```diff
@@ -0,0 +1,160 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.spanner.test;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import org.springframework.cloud.gcp.data.spanner.core.mapping.SpannerColumn;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.SpannerPersistentEntity;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.SpannerTable;
+import org.springframework.data.annotation.Id;
+
+/**
+ * @author Ray Tsang
+ * @author Balint Pato
+ * @author Chengyuan Zhao
+ */
+@SpannerTable(name = "#{'trades_'.concat(tablePostfix)}")
```

Comment:
`tablePostfix` isn't a really intuitive name, unless I'm missing anything.

Reply:
I mean the variable `tablePostfix`. I have no idea what it means. Am I missing on some obscure CS lingo?

Reply:
I think if you rename it "suffix" then it will clear up the confusion.

Reply:
Sounds like this is a domain class. Shouldn't it be moved to the `domain` then ?

Reply:
Hmmm...okay, it might be a language issue indeed, although suffix and postfix are somewhat interchangeable suffix [is used more](https://english.stackexchange.com/questions/81263/postfix-or-suffix) and might be more accurate contextually.

Reply:
Apologies for my poor English :joy: I studied English in a language school for 10 years and have spoken it for more than 20, it's the first time I've ever heard the word haha